torchao/quantization/quant_api.py

```diff
@@ -1588,6 +1596,26 @@ def _fp8_mm_compat(weight: torch.Tensor) -> bool:
     return is_compatible
 
 
+@dataclass
+class Float8TensorConfig(AOBaseTensorConfig):
+    dtype: torch.dtype = e4m3_dtype
+    granularity: FP8Granularity = PerRow
```

Comment:
nit: small doc block here

Reply:
also the default looks weird, since it is a class type not an instance? 
https://github.com/pytorch/ao/blob/2e2ce0b874f62e336dfe7ab40a2f824495b4dd40/torchao/quantization/quant_api.py#L1611

probs needs to be optonal

Reply:
oh right, that's a typo, it seems Optional is not really needed, we need a default anyways right